apps/web/src/app/admin-console/organizations/members/components/member-dialog/member-dialog.component.html

```diff
@@ -262,7 +262,11 @@ <h3 class="tw-mt-4">
           {{ "revokeAccess" | i18n }}
         </button>
         <button
-          *ngIf="editMode"
+          *ngIf="
+            editMode &&
+            ((accountDeprovisioningEnabled$ | async) === false ||
+              params.managedByOrganization === false)
```

Comment:
`accountDeprovisioningEnabled$ | async` and `manangedByOrganization' here will be booleans, I don't think we need the === check. We should be able to just use their values for these conditions:

```suggestion
            editMode &&
           (!(accountDeprovisioningEnabled$ | async) || !params.managedByOrganization)
```

Reply:
Makes sense, I've made that change across all conditions